src/plasma/test/manager_tests.c

```diff
@@ -15,6 +15,11 @@
 #include "plasma_client.h"
 #include "plasma_manager.h"
 
+#ifndef _WIN32
```

Comment:
Can we put this into "win_compat.h" and share it with the same one above? Let's try to avoid having any #ifdef WIN or #ifdef linux in the main code.